### PR TITLE
Added fdivdelay test.

### PR DESCRIPTION
**Pull request by jpd002**
This is a small test that shows the various delays for FDIV instructions (DIV, SQRT, RSQRT).

### CODE REVIEW

File: tests/vu/lower/fdivdelay.cpp [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+		printf("%s single delayed:\n", name);
+
+		Reset();
+
+		//Set Q to 1.0f
+		Wr(DIV(FIELD_W, VF00, FIELD_W, VF00));
+		Wr(WAITQ());
+
+		//Prepare registers
+		WrImm(0x40800000);
```
Comment (unknownbrackets) [inline tests/vu/lower/fdivdelay.cpp]: In this case I would prefer 4.0f.  It's overloaded for floats and in this case would make things clearer.  And similar for other float consts.

-[Unknown]

**Comment by unknownbrackets** [comment]
Cool, nice.  So are these results consistent between runs?  Or does it sometimes vary in the number of instructions it delays for?  I would hope it doesn't vary.

If it does vary, we might want to structure the test output differently so that an emulator could pass it (exact result match) as long as it delayed by the minimum.

-[Unknown]

**Comment by jpd002** [comment]
It doesn't vary and many games rely on that (placing a Q reading operation 7 instructions after a DIV). 

I'll change the constants to actual floats literals to make it clearer.

**Comment by unknownbrackets** [comment]
Sweet, thanks.

-[Unknown]

**Pull request merged**